crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -42,14 +42,18 @@ use crate::{
     helpers::add_config_version,
 };
 
-use super::types::CreateReq;
+use super::types::{CreateReq, UpdateReq};
 
 pub fn endpoints() -> Scope {
-    Scope::new("").service(create).service(get).service(delete)
+    Scope::new("")
+        .service(create_config)
+        .service(update_config)
+        .service(get)
+        .service(delete)
 }
 
-#[put("/{key}")]
-async fn create(
+#[post("/{key}")]
```

Comment:
can we take in requestbody itself 
as its like that in dimension, type_templates, function

Reply:
yeah I wanted to raise this as well
we should keep the formats/structures consistent across endpoints